content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -26,157 +23,289 @@ Get an overview of PodPresets at
+
+```shell
+kubectl create -f https://k8s.io/examples/podpreset/configmap.yaml
+```
+
+Here is the PodPreset configuration file referencing the ConfigMap:
 
 {{< codenew file="podpreset/allow-db.yaml" >}}
 
-**Pod spec after admission controller:**
+Create the preset that references the ConfigMap:
+
+```shell
+kubectl create -f https://k8s.io/examples/podpreset/allow-db.yaml
+```
+
+And the configuration file containing the definition of an applicable Pod looks like this:
+
+{{< codenew file="podpreset/pod.yaml" >}}
+
+Create the Pod:
+
+```shell
+kubectl create -f https://k8s.io/examples/podpreset/pod.yaml
+```
+
+View the Pod spec altered by the admission controller in order to see the effects of the preset
+having been applied:
+
+```shell
+kubectl get pod website -o yaml
+```
 
 {{< codenew file="podpreset/allow-db-merged.yaml" >}}
 
-## ReplicaSet with Pod Spec Example
+The `DB_PORT` environment variable and the `podpreset.admission.kubernetes.io` annotation of the Pod
+verify that the preset has been applied.
+
+## ReplicaSet with Pod Spec example
 
-The following example shows that only the pod spec is modified by the Pod
-Preset.
+This is an example to show that only Pod specs are modified by Pod presets. Other workload types 
+like ReplicaSets or Deployments are unaffected.
+
+Here is the PodPreset configuration file for this example:
+
+{{< codenew file="podpreset/preset.yaml" >}}
 
-**User submitted ReplicaSet:**
+Create the preset:
+
+```shell
+kubectl apply -f https://k8s.io/examples/podpreset/preset.yaml
+```
+
+Here is the configuration file containing the definition of a ReplicaSet whose deployment will result
+in the creation of three Pods:
 
 {{< codenew file="podpreset/replicaset.yaml" >}}
 
-**Example Pod Preset:**
+Create the ReplicaSet:
 
-{{< codenew file="podpreset/preset.yaml" >}}
+```shell
+kubectl create -f https://k8s.io/examples/podpreset/replicaset.yaml
+```
 
-**Pod spec after admission controller:**
+Verify that the Pods created by the ReplicaSet are running:
 
-Note that the ReplicaSet spec was not changed, users have to check individual pods
-to validate that the PodPreset has been applied.
+```shell
+kubectl get pods
+```
 
-{{< codenew file="podpreset/replicaset-merged.yaml" >}}
+The output shows that the Pods are running:
 
-## Multiple PodPreset Example
+```
+NAME             READY   STATUS    RESTARTS   AGE
+frontend-2l94q   1/1     Running   0          2m18s
+frontend-6vdgn   1/1     Running   0          2m18s
+frontend-jzt4p   1/1     Running   0          2m18s
+```
+
+View the spec of the ReplicaSet:
 
-This is an example to show how a Pod spec is modified by multiple Pod
-Injection Policies.
+```shell
+kubectl get replicasets frontend -o yaml
+```
 
-**User submitted pod spec:**
+Note that the ReplicaSet spec was not changed nor does it contain a
+`podpreset.admission.kubernetes.io` annotation.
+To see the effects of the preset having been applied, individual Pods need to be examined.
```

Comment:
I'd use a note shortcode; for example:
```suggestion
{{< note >}}
The ReplicaSet object's `spec` was not changed, nor does the ReplicaSet contain a
`podpreset.admission.kubernetes.io` annotation. This is because a PodPreset only
applies to Pod objects.

To see the effects of the preset having been applied, you need to look at individual Pods.
{{< /note >}}
```